spec/wor/test_services/some_service.rb

```diff
@@ -33,7 +35,9 @@ def method_without_log_without_block
   def method_without_log_with_block
     get(
       path: '/mypath'
-    ) { |response| JSON.parse(response.body)['array'].reduce('', :+) }
+    ) do |response|
+      JSON.parse(response.body)['array'].reduce('', :+) if response.body.present?
```

Comment:
This line is equal to the line in method `method_with_log_with_block`... You can refactor this, move to another method and avoid make repetition in your code